lib/rubocop/cop/layout/trailing_whitespace.rb

```diff
@@ -34,6 +36,25 @@ def investigate(processed_source)
         def autocorrect(range)
           ->(corrector) { corrector.remove(range) }
         end
+
+        private
+
+        def skip_heredoc_lines?
+          cop_config.fetch('AllowInHeredoc', false)
+        end
+
+        def inside_heredoc?(heredoc_lines, index)
+          heredoc_lines.any? { |r| r.include?(index + 1) }
+        end
+
+        def extract_heredoc_lines(ast)
```

Comment:
`lines` should be probably be `ranges`, given the context. One would normally expect lines to just be strings.